CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# next
```

Comment:
Could you move this under 1.1.1? I've already bumped the version, but not released it.